### PR TITLE
Update SGX enclave manager container image in K8s deployment.

### PR DESCRIPTION
**Pull request by manojgop**
- Update docker image name in K8s SGX enclave manager deployment file.
- Update documentation in Readme file.

Signed-off-by: Manoj Gopalakrishnan <manoj.gopalakrishnan@intel.com>

### CODE REVIEW

File: docker/k8s/README.md [hunk 1]
```diff
@@ -45,27 +46,31 @@ in SGX-SIM and SGX-HW mode.
...
 ## Create Avalon shell pod
 1. Avalon shell pod can be useful to run applications manually.
     ```bash
     kubectl create -f shell.yaml
     ```
-2. After verifying avalon shell pod is running get a shell to the running container
+2. After verifying avalon shell pod is running, get a shell to the running container
     ```bash
-    # Let's say 'avalon-shell-abcde' is the pod created above
+    # Get avalon shell pod name using command
```
Comment (danintel) [inline docker/k8s/README.md]: Change "Get avalon shell pod name" to "Get the Avalon shell pod name"

File: docker/k8s/README.md [hunk 2]
```diff
@@ -45,27 +46,31 @@ in SGX-SIM and SGX-HW mode.
...
 
     > The above command would run the 
-    > [propose_requests.sh](./scripts/propose_requests.sh) as a kubernetes job
+    > [propose_requests.sh](../../scripts/propose_requests.sh) as a kubernetes job
 
 3. Check the request handling logs (the request should be done in several 
    minutes after the proposal)
     ```bash
-    # Let's say 'propose-request-abcde' is the job created above
+    # Get workload request pod name using command
```
Comment (danintel) [inline docker/k8s/README.md]: Change "Get workload" to "Get the workload"

File: docker/k8s/README.md [hunk 3]
```diff
@@ -45,27 +46,31 @@ in SGX-SIM and SGX-HW mode.
...
+    kubectl get pods
+    # Let's say 'propose-request-abcde' is the job pod created above. Check the logs
     kubectl logs propose-request-abcde
 
     # The partial output for the last query should be go as below
```
Comment (danintel) [inline docker/k8s/README.md]: Change "go as below" to "as follows"

File: docker/k8s/README.md [hunk 4]
```diff
@@ -6,7 +6,8 @@ in SGX-SIM and SGX-HW mode.
 
 ## Prerequisites
 - kubectl-v1.15.2
-- a kubernetes cluster
+- a kubernetes cluster. [Minikube] (https://kubernetes.io/docs/setup/learning-environment/minikube/)
+  can be used to setup a single node kubernetes cluster.
```
Comment (danintel) [inline docker/k8s/README.md]: Change "kubernetes" to "Kubernetes" (and below at line 49)

File: docker/k8s/README.md (hunk 1 above)
Comment (manojgop) [reply docker/k8s/README.md]: Done

File: docker/k8s/README.md (hunk 2 above)
Comment (manojgop) [reply docker/k8s/README.md]: Done

File: docker/k8s/README.md (hunk 3 above)
Comment (manojgop) [reply docker/k8s/README.md]: Done

File: docker/k8s/README.md (hunk 4 above)
Comment (manojgop) [reply docker/k8s/README.md]: Done

**Pull request merged**